lint.sh

```diff
@@ -78,7 +78,7 @@ check_bash(){
 }
 
 check_python(){
-  flake8 --exclude=.lintvenv . \
+  flake8 --exclude=.lintvenv,webhooktranslator . \
```

Comment:
This was also required to get lint to pass, as the webhook translator dir contained build dirs that didn't pass flake8. Its not directly related to this patch but included because it will cause problems if you switch branches.